components/hab/src/command/studio/enter.rs

```diff
@@ -150,21 +148,23 @@ mod inner {
               str::FromStr};
 
     const SUDO_CMD: &str = "sudo";
+    const STUDIO_CMD: &str = "hab-studio";
+    const STUDIO_CMD_ENVVAR: &str = "HAB_STUDIO_BINARY";
```

Comment:
These were moved into this sub-module because they're no longer used on Windows, correct?

Reply:
Random aside that I'll mention here, since I don't have a better way to do it... while you're in here, it would be good to remove this code: https://github.com/habitat-sh/habitat/blob/0f3e785046e3eb9dca8cbd3b324d69762e392e59/components/hab/src/command/studio/enter.rs#L200-L202

That's in a module that's only compiled on Linux, so it'll never be called.

Reply:
correct. deleting this - good call!